syntax = "proto3";

package seerep;

message ClassQuery
{
  // uuid of the project to query
  string projectuuid = 1;

  // uuid of the class to query
  string classuuid = 2;
}
